includes/discovery/os/firebox.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+if (starts_with($sysDescr, 'WatchGuard Fireware') || starts_with($sysObjectId, '.1.3.6.1.4.1.3097.1.5')) {
```

Comment:
I honestly don't know why we have firebox/watchguard/fireware.

I think they should all be fireware...